views/create-event.html

```diff
@@ -16,7 +16,116 @@
 	<form id="create-event" method="POST">
```

Comment:
The form and its inputs should be given the correct `class`(es) to use bootstrap form styling.  This is secondary, but it would put us in a better spot for next week.

See: http://getbootstrap.com/css/#forms

Bootstrap is already included.